test/e2e-tests/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: test
+test:
+	@(cd tests && go test)
```

Comment:
For people who might not (want to) have golang installed on their machines, could you also show a target which would run the tests from a local container?

Reply:
I could but is this really a need? We are running tests for a k8s cluster locally I think expecting go to be installed is okay. Especially because if you have docker or podman installed on your machine why would go be a deal breaker?

Reply:
I have rootless podman setup exactly because most everything (including for example `oc` or `rosa`) is run in containers. It's because being involved with multiple projects, the requirements for the versions of compilers and libraries (not just golang) that I'd have to match seem to differ wildly and in conflicting manner, so the most sane way is to just have it all containerized.

Even K8s itself does not require you to have golang installed to build it -- when following https://github.com/kubernetes/kubernetes#to-start-developing-k8s the `make quick-release` works through docker. And with https://github.com/kubernetes/kubernetes/pull/121533 even with podman.

Reply:
If we are standardizing around `gotest` for the test framework it would imply that `go` is available locally.

@jackdelahunt One recommendation I would make is that you replace the call to `go` with a Makefile variable that defaults to `GO = go`.  This would add support for calling a specific`goX.Y.Z` binary version or absolute path 
```
make GO=go1.19 test
```

@adelton I use the same workflow for my local development.  For that use case, I would redirect anyone to [toolbox](https://github.com/containers/toolbox) or [distrobox](https://github.com/89luca89/distrobox) if they want to run those binaries from a containerized environment with the current directory mounted

Reply:
...and can you add a new line to the end of the file

Reply:
Sure sounds good to me 👍